UPGRADE-6.3.md

```diff
@@ -57,6 +57,12 @@ FrameworkBundle
 
  * Deprecate the `notifier.logger_notification_listener` service, use the `notifier.notification_logger_listener` service instead
 
+HttpClient
+----------
+
+* Deprecate the `Http\Client\HttpClient` service, use the `Http\Client\HttpAsyncClient` service for method `::sendAsyncRequest` and `psr18.http_client` for `::sendRequest` instead
```

Comment:
```suggestion
* Deprecate the `Http\Client\HttpClient` service, use the `Http\Client\HttpAsyncClient` service for method `::sendAsyncRequest` and `Psr\Http\Client\ClientInterface` for `::sendRequest` instead
```

Reply:
Good point, somehow forgot the ClientInterface surely is a service alias 👍